src/vm_pmu.rs

```diff
@@ -373,6 +373,112 @@ impl VmPmuState {
         }
     }
 
+    // Implementation of PMU counter configuration on platforms with Sscdeleg support
+    // TODO: Use code in the actual implementation
+    #[allow(dead_code)]
+    fn configure_matching_counters_internal(
```

Comment:
nit: The suffix can be Sscdeleg to be explicit. The SBI function can be sbi as a explicit prefix.